korman/properties/modifiers/anim.py

```diff
@@ -143,6 +143,15 @@ def post_export(self, exporter, bo, so):
             # If the mass is zero, then we will fail to animate. Fix that.
             if phys.mass == 0.0:
                 phys.mass = 1.0
+                
+                # On CC and Prime, set kPinned so it doesn't fall through
+                if bpy.context.scene.world.plasma_age.version in ["pvPots", "pvPrime"]:
```

Comment:
Do this regardless of version--the behavior may change with a physics engine change.